q2_health_index/_gmhi.py

```diff
@@ -47,10 +48,55 @@ def calculate_gmhi(ctx,
 
     # TODO Valentyn: CALCULATE GMHI (BELOW WE ARE JUST LOADING EXPECTED DATA !!!!)
     expected = os.path.join(os.path.dirname(__file__), 'tests/data/expected/4347_final_gmhi.tsv')
-    gmhi = pd.read_csv(expected, sep='\t', index_col=0, header=0, squeeze=True)
+    gmhi_in = pd.read_csv(expected, sep='\t', index_col=0, header=0, squeeze=True)
+
+    # Removing unclassified and virus species - skipped transposing steps
+    na_species = gmhi_in.index.str.contains('unclassified|virus', regex=True)
+    species_profile_2 = gmhi_in[~na_species]
+
+    # Re-normalization of species' relative abundances after removing unclassified and virus species
+    species_profile_3 = species_profile_2.apply(lambda x: x / x.sum(), axis=0)
+    species_profile_3[species_profile_3 < 0.00001] = 0
+
+    # Extracting Health-prevalent species present in metagenome
+    MH_species = species_profile_3[species_profile_3.index.isin(healthy_species_list)]
```

Comment:
what does `MH` stand for?
I would find the code easier to read, if the variables were more verbose

Reply:
I kept it this way for 2 reasons: 
1. To make it easy to follow when comparing to the methodology section of the original article methods section. This `MH_species` refers to health-associated species of gut bacteria.
> These species are included as an element of “Health-prevalent” species MH, or the set of species more frequently observed in group H than in group N.
2. To make it easy to compare to original code - in case the logic of our software will break during development we can easily refer to original source. 

After testing is finished I will proceed with the more verbose variable assignment.